docs/guide/ecosystem/documentation.md

```diff
@@ -151,38 +155,31 @@ props: {
 
 #### Pros
 
-- Displays a live editor to try components
-- Meant for design system creation (powers vue-design-system)
-- A strong React community
+- Displays a live editor to try components even on the static website
+- Meant for design system creation and showcase (powers vue-design-system)
+- Multiple components examples can be shown in the same markdown page
 
 #### Cons
 
-- Little configurability on output (HTML only)
+- Generated output is currently limited to just HTML, using a cussopredefined template.
 - JSDoc can be verbose
 - Scaffolding built in React
 
-### Vuese
-
-https://github.com/vuese/vuese
-
-Vuese is a documentation oriented.
-It sacrifices a bit of user experience for a lot of integrability.
-If you have to generate your doc to integrate it, this is probably the tool that you want.
+#### Useful Links
 
-Example: https://buptsteve.github.io/markdown-it-vuese/example/demo.html#props
+- [Github](https://github.com/vue-styleguidist/vue-styleguidist)
+- [Example](http://rafaelescala.com/vue-styleguide/)
+- [Documentation](https://vue-styleguidist.github.io/)
+- [Discord Channel](https://discordapp.com/channels/325477692906536972/538786416092512278)
 
-It supports all the following vue features:
+### Vuese
 
-- Single file components
-- Class style components
-- Template and script commenting (Events, Methods, Props, Slots)
-- JSX & TSX
-- TypeScript & Flow
-- Vuex & Vue-i18n
+[Vuese](https://github.com/vuese/vuese) is documentation oriented. It sacrifices interactivity to be more integrable.
+If you have to generate your doc to integrate it in a bigger one, this is the workbench that you want. Without sacrificing the Hot Module Reload, it generates a good looking static doc.
 
 #### Documentation style
 
-vuese uses its own documentaion format more compact than JSDoc
+Vuese uses its own documentation format, more compact than JSDoc.
 
 ```javascript
 props: {
```

Comment:
Lets expand the example a bit more, like here 
https://vuese.github.io/vuese-explorer/